fastlane_bot/tests/deterministic/_data/test_strategies.json

```diff
@@ -0,0 +1,56 @@
+{
+    "test_strategies": {
+        "1": {
+            "token0": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+            "token1": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+            "y0": 1267460000029073000,
+            "z0": 1267460000029073000,
+            "A0": 0,
+            "B0": 4411844567835374,
+            "y1": 7634017977,
+            "z1": 7649999999,
+            "A1": 0,
+            "B1": 11433567603,
+            "wallet": "0x28C6c06298d514Db089934071355E5743bf21d60"
+        },
+        "2": {
+            "token0": "0x514910771AF9Ca656af840dff83E8264EcF986CA",
+            "token1": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+            "y0": 80000000000000000000,
+            "z0": 80000000000000000000,
+            "A0": 0,
+            "B0": 1399247190132272,
+            "y1": 0,
+            "z1": 0,
+            "A1": 0,
+            "B1": 17102934719008,
+            "wallet": "0x28C6c06298d514Db089934071355E5743bf21d60"
+        },
+        "3": {
+            "token0": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+            "token1": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+            "y0": 3000000000000000000000,
+            "z0": 3000000000000000000000,
+            "A0": 0,
+            "B0": 6052452418541427,
+            "y1": 0,
+            "z1": 0,
+            "A1": 0,
+            "B1": 0,
+            "wallet": "0x28C6c06298d514Db089934071355E5743bf21d60"
+        },
+        "4": {
+            "token0": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
+            "token1": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+            "y0": 100000000,
+            "z0": 100000000,
+            "A0": 17694999,
+            "B0": 645747883,
+            "y1": 0,
+            "z1": 0,
+            "A1": 0,
+            "B1": 0,
+            "wallet": "0x28C6c06298d514Db089934071355E5743bf21d60"
+        }
+    }
+}
```

Comment:
Best to maintain all floating point values as strings, and convert them to Decimal in the code, so as to avoid accuracy loss.

Reply:
can you be more specific about which values you're referring to? as of now, the tests pass in the sense of finding the expected values, and what you're suggesting seems like it could mean converting all float and int types to Decimal, which is a fairly risky effort in terms of introducing bugs -- which in this case means attempting to fix what isn't broken

Reply:
I'm saying that in data files (JSONs CSVs, etc), we should generally represent non-integer values with strings.